.env.example

```
@@ -62,6 +66,9 @@ JWT_TOKEN="your_jwt_token_here"
 USER_EMAIL='user@example.com'
 USER_PASSWORD='your_super_secret_password_here'
 
+# Gemini OAuth Cache Directory
+GEMINI_OAUTH_CACHE='~/.gemini/oauth_creds.json'
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The GEMINI_OAUTH_CACHE key is duplicated

(DuplicatedKey)